index.html

```diff
@@ -29,5 +29,122 @@ <h4>CONNECT WITH US</h4>
+                <li>javascript</li>
+            </ul>
+            <button class="button" type="button">See Project</button>
+            </div>        
+        </div>
+
+        <div class="work-1">
+            <div class="work-img"><img src="./assets/images/Snapshoot-Portfolio3.png" alt="Portfolio snapshoot"></div>
+            <div class="work-desc"><h1>Multi-Post
+                Stories</h1>
+            <ul class="frame">
+                <li class="canopy">CANOPY</li>
+                <li>Back End Dev</li>
+                <li>2022</li>
+            </ul>
+            <p  class="work1-p">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="framework">
+                <li>html</li>
+                <li>css</li>
+                <li>javascript</li>
+            </ul>
+            <button class="button" type="button">See Project</button>
+            </div>        
+        </div>
+        
+        <section class="description-1">
+            <h2>About<br>
+                Myself</h2>
+            <p>I’m a software developer! I can help you develop a product , feature or website Look through some of my work and experience! If you like what you see and have a project or content you to need create, don’t hestiate to contact me.</p>
+            <h4>CONNECT WITH US</h4>
+            <ul class="socials">
+                 <li><a href="#"><i class="fab fa-twitter"></i></a></li>
+                 <li><a href="#"><i class="fab fa-linkedin"></i></a></li>
+                <li><a href="#"><i class="fab fa-facebook"></i></a></li>
+                <li><a href="#"><i class="fab fa-github"></i></a></li>
+                <li><a href="#"><i class="fab fa-medium-m"></i></a></li>  
+            </ul>
+            <button class="button" type="button">Get my resume</button>
+            <ul class="first language">
+                <li ><h4>Languages</h4></li>
+                <li><i class="fas fa-angle-down"></i></li>
+            </ul>
+                <ul class="list">
+                    <li class="lang">
+                        <img src="./assets/images/Ellipse-1.png" alt="js">
+                        <p>JavaScript</p>
+                    </li>
+                    <li class="lang">
+                        <img src="./assets/images/Ellipse-2.png" alt="html">
+                        <p>HTML</p>
+                    </li>
+                    <li class="lang">
+                        <img src="./assets/images/Ellipse-3.png" alt="css">
+                        <p>CSS</p>
+                    </li>
+                </ul>
+                <ul class="language">
+                    <li ><h4>Framework</h4></li>
+                    <li><i class="fas fa-angle-right"></i></li>
+                </ul>
+                <ul class="language">
+                    <li ><h4>Skills</h4></li>
+                    <li><i class="fas fa-angle-right"></i></li>
+                </ul>    
+        </section>
```

Comment:
[OPTIONAL]
- You have implemented what the previous reviewer said. That's perfect. Let's improve our code by moving the about section out of the works section so our code is well organized. try to use the following structure:
```html
<section class="works-grid">
   <div class="work-1">
   <div class="work-1">
  ...
</section>
<section class="description-1">
...
</section>